src/engine/controls/loopingcontrol.cpp

```diff
@@ -1307,7 +1307,9 @@ double LoopingControl::findBeatloopSizeForLoop(
             }
         }
     }
-    return -1;
+
+    // No hit. Calculate the fractional beat length
+    return pBeats->numFractionalBeatsInRange(startPosition, endPosition);
```

Comment:
I wonder why numBeatsInRange() does not returns factions in the first place. 
It could be also required here: 
https://github.com/mixxxdj/mixxx/blob/fe0c53030ca56563dc6b07bbff3923105d9ddb45/src/engine/controls/loopingcontrol.cpp#L768

Reply:
This branch is for when quantize is enabled, we don't need fractional lengths then.